### PR TITLE
Expose stream level and flags

### PR DESCRIPTION
**Pull request by magicbear**
Expose stream level and flags for remuxing stream.

### CODE REVIEW

File: av/packet.pyx [hunk 1]
```diff
@@ -204,6 +204,9 @@ cdef class Packet(Buffer):
             if self.struct.duration != lib.AV_NOPTS_VALUE:
                 return self.struct.duration
 
+    property flags:
```
Comment (jlaine) [inline av/packet.pyx]: For new code, we prefer the modern form:

```
@property
def flags(self):
    return self.struct.flags
```

Comment (jlaine) [reply av/packet.pyx]: Also, @mikeboers has some infrastructure in place for enums, would it be appropriate to use these?

My concern is that simply exposing "flags" as-is means that users will have to hard-code the AV_PKT_FLAG_xx in their code to interpret "flags".

Comment (magicbear) [reply av/packet.pyx]: I use the flags for remux the stream, just copy it, so using the original flags value

**Comment by jlaine** [comment]
The CI build failure is not due to your changes, there is an issue with OS X builds

**Comment by WyattBlue** [comment]
Closing this because of its age and lack of interest

**Pull request closed without merging**